src/tokens.md

```diff
@@ -93,7 +93,7 @@ A suffix is a sequence of characters following the primary part of a literal (wi
 
 > **<sup>Lexer</sup>**\
 > SUFFIX : IDENTIFIER_OR_KEYWORD\
-> SUFFIX_NO_E : SUFFIX <sub>_not beginning with `e` or `E`_</sub>
+> SUFFIX_NO_E : SUFFIX <sub>_not beginning with `e` or `E`_</sub>
```

Comment:
I don't see the change here. Tab/space?

Reply:
It is the no-break-space character (0xa0). I do not think it is particularly necessary here. If we really want to use no-break-space, then the invisible characters should be replaced with `&nbsp;` so it is visible in an editor.